### ISSUE TITLE
[Nightly Build] kubelet failed to update node status

### ISSUE THREAD

**Issue by divyenpatel** [vmware/kubernetes#113]
Nightly CI Build: v1.7.0-alpha.0.805+2d319bd4061c55
hyperkube: https://storage.googleapis.com/kubernetes-release-dev/ci/v1.7.0-alpha.0.805+2d319bd4061c55/bin/linux/amd64/hyperkube

Deployment Log: http://10.126.241.1:8080/job/k8s-release-dev-ci-latest/4/consoleFull

```
root@master [ ~ ]# ./kubectl version                
Client Version: version.Info{Major:"1", Minor:"7+", GitVersion:"v1.7.0-alpha.0.805+2d319bd4061c55", GitCommit:"2d319bd4061c55dcc610cd3e7c8137d9fe95affc", GitTreeState:"clean", BuildDate:"2017-03-04T05:19:38Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"7+", GitVersion:"v1.7.0-alpha.0.805+2d319bd4061c55", GitCommit:"2d319bd4061c55dcc610cd3e7c8137d9fe95affc", GitTreeState:"clean", BuildDate:"2017-03-04T05:15:06Z", GoVersion:"go1.7.5", Compiler:"gc", Platform:"linux/amd64"}
root@master [ ~ ]# 
```

```
root@master [ ~ ]# ./kubectl get nodes
NAME      STATUS                     AGE       VERSION
master    Ready,SchedulingDisabled   4h        v1.7.0-alpha.0.805+2d319bd4061c55
node1     NotReady                   4h        v1.7.0-alpha.0.805+2d319bd4061c55
node2     NotReady                   4h        v1.7.0-alpha.0.805+2d319bd4061c55
node3     Ready                      4h        v1.7.0-alpha.0.805+2d319bd4061c55
node4     NotReady                   4h        v1.7.0-alpha.0.805+2d319bd4061c55
root@master [ ~ ]#
```

```
root@master [ ~ ]# ./kubectl describe node node1
Name:			node1
Role:			
Labels:			alpha.kubernetes.io/fluentd-ds-ready=true
			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node1
Annotations:		node.alpha.kubernetes.io/ttl=0
			volumes.kubernetes.io/controller-managed-attach-detach=true
Taints:			<none>
CreationTimestamp:	Sat, 04 Mar 2017 07:12:57 +0000
Phase:			
Conditions:
  Type			Status		LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------		-----------------			------------------			------			-------
  OutOfDisk 		Unknown 	Sat, 04 Mar 2017 07:13:28 +0000 	Sat, 04 Mar 2017 07:14:12 +0000 	NodeStatusUnknown 	Kubelet stopped posting node status.
  MemoryPressure 	Unknown 	Sat, 04 Mar 2017 07:13:28 +0000 	Sat, 04 Mar 2017 07:14:12 +0000 	NodeStatusUnknown 	Kubelet stopped posting node status.
  DiskPressure 		Unknown 	Sat, 04 Mar 2017 07:13:28 +0000 	Sat, 04 Mar 2017 07:14:12 +0000 	NodeStatusUnknown 	Kubelet stopped posting node status.
  Ready 		Unknown 	Sat, 04 Mar 2017 07:13:28 +0000 	Sat, 04 Mar 2017 07:14:12 +0000 	NodeStatusUnknown 	Kubelet stopped posting node status.
Addresses:		10.20.104.54,fe80::250:56ff:fe8e:a16a,fc00:10:20:107:250:56ff:fe8e:a16a,fc00:10:20:106:250:56ff:fe8e:a16a,fc00:10:20:105:250:56ff:fe8e:a16a,fc00:10:20:104:250:56ff:fe8e:a16a,172.1.9.1,node1
Capacity:
 cpu:		2
 memory:	4048416Ki
 pods:		110
Allocatable:
 cpu:		2
 memory:	3946016Ki
 pods:		110
System Info:
 Machine ID:			efee03ac51c6418889650dfa2a40350d
 System UUID:			420EA2A3-10B6-48BE-D129-6855F944FDDB
 Boot ID:			00c988b9-17c6-4f76-9f90-1fe817c2b730
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://0.1.0
 Kubelet Version:		v1.7.0-alpha.0.805+2d319bd4061c55
 Kube-Proxy Version:		v1.7.0-alpha.0.805+2d319bd4061c55
PodCIDR:			10.244.4.0/24
ExternalID:			/Datacenter/vm/kubernetes/node1
Non-terminated Pods:		(2 in total)
  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----				------------	----------	---------------	-------------
  kube-system			kube-dns-v19-d0n4d		110m (5%)	0 (0%)		120Mi (3%)	220Mi (5%)
  kube-system			kube-proxy-jrln3		100m (5%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  210m (10%)	0 (0%)		120Mi (3%)	220Mi (5%)
Events:
  FirstSeen	LastSeen	Count	From		SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----		-------------	--------	------			-------
  4h		7s		8193	kubelet, node1			Normal		NodeHasSufficientDisk	Node node1 status is now: NodeHasSufficientDisk
  4h		7s		8193	kubelet, node1			Normal		NodeHasSufficientMemory	Node node1 status is now: NodeHasSufficientMemory
  4h		7s		8193	kubelet, node1			Normal		NodeHasNoDiskPressure	Node node1 status is now: NodeHasNoDiskPressure
  4h		7s		8185	kubelet, node1			Normal		NodeReady		Node node1 status is now: NodeReady
root@master [ ~ ]#

root@master [ ~ ]# ./kubectl describe node node3
Name:			node3
Role:			
Labels:			alpha.kubernetes.io/fluentd-ds-ready=true
			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node3
Annotations:		node.alpha.kubernetes.io/ttl=0
			volumes.kubernetes.io/controller-managed-attach-detach=true
Taints:			<none>
CreationTimestamp:	Sat, 04 Mar 2017 07:12:57 +0000
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason				Message
  ----			------	-----------------			------------------			------				-------
  OutOfDisk 		False 	Sat, 04 Mar 2017 12:07:52 +0000 	Sat, 04 Mar 2017 07:12:57 +0000 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Sat, 04 Mar 2017 12:07:52 +0000 	Sat, 04 Mar 2017 07:12:57 +0000 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Sat, 04 Mar 2017 12:07:52 +0000 	Sat, 04 Mar 2017 07:12:57 +0000 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Sat, 04 Mar 2017 12:07:52 +0000 	Sat, 04 Mar 2017 07:12:57 +0000 	KubeletReady 			kubelet is posting ready status
Addresses:		10.20.104.66,fe80::250:56ff:fe8e:df52,fc00:10:20:107:250:56ff:fe8e:df52,fc00:10:20:106:250:56ff:fe8e:df52,fc00:10:20:105:250:56ff:fe8e:df52,fc00:10:20:104:250:56ff:fe8e:df52,172.1.1.1,node3
Capacity:
 cpu:		2
 memory:	4048416Ki
 pods:		110
Allocatable:
 cpu:		2
 memory:	3946016Ki
 pods:		110
System Info:
 Machine ID:			efee03ac51c6418889650dfa2a40350d
 System UUID:			420EB28A-EA93-F899-CEA9-1ADF5D94AE9E
 Boot ID:			c8b436f3-3c75-4ea3-a1d0-18e4f48d830a
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://0.1.0
 Kubelet Version:		v1.7.0-alpha.0.805+2d319bd4061c55
 Kube-Proxy Version:		v1.7.0-alpha.0.805+2d319bd4061c55
PodCIDR:			10.244.0.0/24
ExternalID:			/Datacenter/vm/kubernetes/node3
Non-terminated Pods:		(1 in total)
  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----				------------	----------	---------------	-------------
  kube-system			kube-proxy-kbvnd		100m (5%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.)
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  100m (5%)	0 (0%)		0 (0%)		0 (0%)
Events:		<none>
root@master [ ~ ]#
```

```
root@node1 [ ~ ]# systemctl -l status kubelet
● kubelet.service - Kubernetes Kubelet Server
   Loaded: loaded (/etc/systemd/system/kubelet.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2017-03-04 07:11:18 UTC; 5h 49min ago
     Docs: https://github.com/kubernetes/kubernetes
  Process: 596 ExecStartPre=/bin/mount --make-shared /var/lib/kubelet (code=exited, status=0/SUCCESS)
  Process: 592 ExecStartPre=/bin/mount --bind /var/lib/kubelet /var/lib/kubelet (code=exited, status=0/SUCCESS)
  Process: 588 ExecStartPre=/bin/mkdir -p /var/lib/kubelet (code=exited, status=0/SUCCESS)
 Main PID: 600 (docker)
   CGroup: /system.slice/kubelet.service
           └─600 /usr/bin/docker run --net=host --pid=host --privileged -v /dev:/dev -v /sys:/sys:ro -v /var/run:/var/run:rw -v /var/lib/docker/:/var/lib/docker:rw -v /var/lib/kubelet/:/var/lib/kubelet:shared -v /var/log:/var/log:shared -v /srv/kubernetes:/srv/kubernetes:ro -v /etc/kubernetes:/etc/kubernetes:ro docker.io/divyen/hyperkube-amd64:ci-green-nightly /hyperkube kubelet --address=0.0.0.0 --allow-privileged=true --cloud-provider=vsphere --enable-server --enable-debugging-handlers --kubeconfig=/srv/kubernetes/kubeconfig.json --pod-manifest-path=/etc/kubernetes/manifests --cluster-dns=10.0.0.10 --cluster-domain=cluster.local --v=2 --api-servers=https://10.20.105.115 --hairpin-mode=promiscuous-bridge --cloud-config=/etc/kubernetes/vsphere.conf

Mar 04 13:01:11 node1 docker[600]: I0304 13:01:11.893660     707 kubelet_node_status.go:380] Recording NodeReady event message for node node1
Mar 04 13:01:11 node1 docker[600]: E0304 13:01:11.901746     707 kubelet_node_status.go:327] Error updating node status, will retry: failed to patch status "{\"status\":{\"addresses\":[{\"address\":\"fe80::9099:c9ff:fe5c:15f8\",\"type\":\"ExternalIP\"},{\"address\":\"fe80::42:8cff:fe4d:b91f\",\"type\":\"ExternalIP\"},{\"address\":\"172.1.9.1\",\"type\":\"ExternalIP\"},{\"address\":\"fc00:10:20:106:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:107:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fe80::250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"10.20.104.54\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:104:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:105:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"}],\"conditions\":[{\"lastHeartbeatTime\":\"2017-03-04T13:01:11Z\",\"lastTransitionTime\":\"2017-03-04T13:01:11Z\",\"message\":\"kubelet has no disk pressure\",\"reason\":\"KubeletHasNoDiskPressure\",\"status\":\"False\",\"type\":\"DiskPressure\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:11Z\",\"lastTransitionTime\":\"2017-03-04T13:01:11Z\",\"message\":\"kubelet has sufficient memory available\",\"reason\":\"KubeletHasSufficientMemory\",\"status\":\"False\",\"type\":\"MemoryPressure\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:11Z\",\"lastTransitionTime\":\"2017-03-04T13:01:11Z\",\"message\":\"kubelet has sufficient disk space available\",\"reason\":\"KubeletHasSufficientDisk\",\"status\":\"False\",\"type\":\"OutOfDisk\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:11Z\",\"lastTransitionTime\":\"2017-03-04T13:01:11Z\",\"message\":\"kubelet is posting ready status\",\"reason\":\"KubeletReady\",\"status\":\"True\",\"type\":\"Ready\"}],\"images\":[{\"names\":[\"divyen/hyperkube-amd64:ci-green-nightly\"],\"sizeBytes\":595263030},{\"names\":[\"gcr.io/google_containers/kubedns-amd64:1.7\"],\"sizeBytes\":55064828},{\"names\":[\"cnastorage/k8s-ignition:nightly\"],\"sizeBytes\":29559981},{\"names\":[\"gcr.io/google_containers/exechealthz-amd64:1.2\"],\"sizeBytes\":8374840},{\"n
Mar 04 13:01:11 node1 docker[600]: ames\":[\"gcr.io/google_containers/kube-dnsmasq-amd64:1.3\"],\"sizeBytes\":5125973},{\"names\":[\"gcr.io/google_containers/pause-amd64:3.0\"],\"sizeBytes\":746888}]}}" for node "node1": Node "node1" is invalid: status.addresses[4]: Duplicate value: api.NodeAddress{Type:"InternalIP", Address:"fc00:10:20:105:250:56ff:fe8e:a16a"}
Mar 04 13:01:12 node1 docker[600]: I0304 13:01:12.031268     707 kubelet_node_status.go:380] Recording NodeHasSufficientDisk event message for node node1
Mar 04 13:01:12 node1 docker[600]: I0304 13:01:12.031329     707 kubelet_node_status.go:380] Recording NodeHasSufficientMemory event message for node node1
Mar 04 13:01:12 node1 docker[600]: I0304 13:01:12.032768     707 kubelet_node_status.go:380] Recording NodeHasNoDiskPressure event message for node node1
Mar 04 13:01:12 node1 docker[600]: I0304 13:01:12.032818     707 kubelet_node_status.go:380] Recording NodeReady event message for node node1
Mar 04 13:01:12 node1 docker[600]: E0304 13:01:12.040865     707 kubelet_node_status.go:327] Error updating node status, will retry: failed to patch status "{\"status\":{\"addresses\":[{\"address\":\"fe80::9099:c9ff:fe5c:15f8\",\"type\":\"ExternalIP\"},{\"address\":\"fe80::42:8cff:fe4d:b91f\",\"type\":\"ExternalIP\"},{\"address\":\"172.1.9.1\",\"type\":\"ExternalIP\"},{\"address\":\"fc00:10:20:106:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:107:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fe80::250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"10.20.104.54\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:104:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"},{\"address\":\"fc00:10:20:105:250:56ff:fe8e:a16a\",\"type\":\"InternalIP\"}],\"conditions\":[{\"lastHeartbeatTime\":\"2017-03-04T13:01:12Z\",\"lastTransitionTime\":\"2017-03-04T13:01:12Z\",\"message\":\"kubelet has no disk pressure\",\"reason\":\"KubeletHasNoDiskPressure\",\"status\":\"False\",\"type\":\"DiskPressure\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:12Z\",\"lastTransitionTime\":\"2017-03-04T13:01:12Z\",\"message\":\"kubelet has sufficient memory available\",\"reason\":\"KubeletHasSufficientMemory\",\"status\":\"False\",\"type\":\"MemoryPressure\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:12Z\",\"lastTransitionTime\":\"2017-03-04T13:01:12Z\",\"message\":\"kubelet has sufficient disk space available\",\"reason\":\"KubeletHasSufficientDisk\",\"status\":\"False\",\"type\":\"OutOfDisk\"},{\"lastHeartbeatTime\":\"2017-03-04T13:01:12Z\",\"lastTransitionTime\":\"2017-03-04T13:01:12Z\",\"message\":\"kubelet is posting ready status\",\"reason\":\"KubeletReady\",\"status\":\"True\",\"type\":\"Ready\"}],\"images\":[{\"names\":[\"divyen/hyperkube-amd64:ci-green-nightly\"],\"sizeBytes\":595263030},{\"names\":[\"gcr.io/google_containers/kubedns-amd64:1.7\"],\"sizeBytes\":55064828},{\"names\":[\"cnastorage/k8s-ignition:nightly\"],\"sizeBytes\":29559981},{\"names\":[\"gcr.io/google_containers/exechealthz-amd64:1.2\"],\"sizeBytes\":8374840},{\"n
Mar 04 13:01:12 node1 docker[600]: ames\":[\"gcr.io/google_containers/kube-dnsmasq-amd64:1.3\"],\"sizeBytes\":5125973},{\"names\":[\"gcr.io/google_containers/pause-amd64:3.0\"],\"sizeBytes\":746888}]}}" for node "node1": Node "node1" is invalid: status.addresses[4]: Duplicate value: api.NodeAddress{Type:"InternalIP", Address:"fc00:10:20:105:250:56ff:fe8e:a16a"}
Mar 04 13:01:12 node1 docker[600]: E0304 13:01:12.040901     707 kubelet_node_status.go:319] Unable to update node status: update node status exceeds retry count
root@node1 [ ~ ]#
```

**Comment by divyenpatel** [comment]
Logs
-----
[node2-logs.zip](https://github.com/vmware/kubernetes/files/819115/node2-logs.zip)
[node4-log.zip](https://github.com/vmware/kubernetes/files/819113/node4-log.zip)
[node3-log.zip](https://github.com/vmware/kubernetes/files/819117/node3-log.zip)
[node1-log.zip](https://github.com/vmware/kubernetes/files/819114/node1-log.zip)
[master-logs.zip](https://github.com/vmware/kubernetes/files/819116/master-logs.zip)

**Comment by divyenpatel** [comment]
@abrarshivani observed this from `systemctl -l status kubelet` on node1
for node "node1": Node "node1" is invalid: status.addresses[4]: Duplicate value: api.NodeAddress{Type:"InternalIP", Address:"fc00:10:20:105:250:56ff:fe8e:a16a"}

**Comment by divyenpatel** [comment]
Kubernetes Issue : https://github.com/kubernetes/kubernetes/issues/42690

**Comment by BaluDontu** [comment]
Duplicate of #103

**Comment by divyenpatel** [comment]
**Build: v1.7.0-alpha.0.1464+a84f100faa927b**

Regarding this issue.
I just set `DHCP=ipv4` in all nodes and rebooted them.

```
# cat /etc/systemd/network/10-dhcp-en.network 
[Match]
Name=e*

[Network]
DHCP=ipv4
```
This temporarily unblocks us for testing 1.7 build.
I was able to create storage class, pvc, pv and pod using `storage.k8s.io/v1 `APIs.


```
$ kubectl version --short
Client Version: v1.5.3
Server Version: v1.7.0-alpha.0.1464+a84f100faa927b
```


```
$ kubectl get nodes
NAME      STATUS                     AGE
master    Ready,SchedulingDisabled   9h
node1     Ready                      9h
node2     Ready                      9h
node3     Ready                      9h
node4     Ready                      9h
```

```
$ kubectl get pods
NAME           READY     STATUS    RESTARTS   AGE
thinclaimpod   1/1       Running   0          1m
```


```
$ kubectl get pvc
NAME        STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
thinclaim   Bound     pvc-67b758ac-0fe9-11e7-866e-0050568a0ce0   2Gi        RWO           2m
```

```
$ kubectl get pv
NAME                                       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS    CLAIM               REASON    AGE
pvc-67b758ac-0fe9-11e7-866e-0050568a0ce0   2Gi        RWO           Delete          Bound     default/thinclaim             2m
```

```
$ kubectl get storageclass
NAME      KIND
thin      StorageClass.v1.storage.k8s.io
divyenp-m01:~ divyenp$ 
```

```
$ kubectl describe nodes
Name:			master
Role:			
Labels:			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=master
Taints:			<none>
CreationTimestamp:	Thu, 23 Mar 2017 00:15:10 -0700
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------	-----------------			------------------			------			-------
  OutOfDisk 		False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 00:15:10 -0700 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 00:15:10 -0700 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 00:15:10 -0700 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 00:15:10 -0700 	KubeletReady 		kubelet is posting ready status
Addresses:		172.1.67.1,master,10.162.45.124
Capacity:
 cpu:		4
 memory:	8170804Ki
 pods:		110
Allocatable:
 cpu:		4
 memory:	8068404Ki
 pods:		110
System Info:
 Machine ID:			8e025a21a4254e11b028584d9d8b12c4
 System UUID:			420A05FC-959D-8FC3-E7FB-6B637DD2462C
 Boot ID:			f944f5c8-26b1-45ae-9d16-5b49650ebf2d
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://1.11.0
 Kubelet Version:		v1.7.0-alpha.0.1464+a84f100faa927b
 Kube-Proxy Version:		v1.7.0-alpha.0.1464+a84f100faa927b
PodCIDR:			10.244.0.0/24
ExternalID:			/vcqaDC/vm/kubernetes/master
Non-terminated Pods:		(5 in total)
  Namespace			Name					CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----					------------	----------	---------------	-------------
  kube-system			etcd-server-master			200m (5%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			kube-apiserver-master			250m (6%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			kube-controller-manager-master		200m (5%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			kube-proxy-qpk22			100m (2%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			kube-scheduler-master			100m (2%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  850m (21%)	0 (0%)		0 (0%)		0 (0%)
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  15m		15m		1	{kubelet master}			Normal		Starting		Starting kubelet.
  15m		15m		1	{kubelet master}			Warning		ImageGCFailed		unable to find data for container /
  15m		15m		1	{kubelet master}			Normal		NodeNotSchedulable	Node master status is now: NodeNotSchedulable
  15m		15m		1	{kube-proxy master}			Normal		Starting		Starting kube-proxy.
  15m		15m		24	{kubelet master}			Normal		NodeHasSufficientDisk	Node master status is now: NodeHasSufficientDisk
  15m		15m		24	{kubelet master}			Normal		NodeHasSufficientMemory	Node master status is now: NodeHasSufficientMemory
  15m		15m		24	{kubelet master}			Normal		NodeHasNoDiskPressure	Node master status is now: NodeHasNoDiskPressure
  15m		15m		1	{kubelet master}			Warning		Rebooted		Node master has been rebooted, boot id: f944f5c8-26b1-45ae-9d16-5b49650ebf2d


Name:			node1
Role:			
Labels:			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node1
Taints:			<none>
CreationTimestamp:	Thu, 23 Mar 2017 00:17:18 -0700
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------	-----------------			------------------			------			-------
  OutOfDisk 		False 	Thu, 23 Mar 2017 10:02:48 -0700 	Thu, 23 Mar 2017 00:17:18 -0700 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Thu, 23 Mar 2017 10:02:48 -0700 	Thu, 23 Mar 2017 00:17:18 -0700 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Thu, 23 Mar 2017 10:02:48 -0700 	Thu, 23 Mar 2017 00:17:18 -0700 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Thu, 23 Mar 2017 10:02:48 -0700 	Thu, 23 Mar 2017 00:17:18 -0700 	KubeletReady 		kubelet is posting ready status
Addresses:		fe80::42:32ff:feb0:4c01,node1,fd01:0:101:2617:250:56ff:fe8a:8eb8
Capacity:
 cpu:		4
 memory:	8170804Ki
 pods:		110
Allocatable:
 cpu:		4
 memory:	8068404Ki
 pods:		110
System Info:
 Machine ID:			8e025a21a4254e11b028584d9d8b12c4
 System UUID:			420A5C51-B112-0735-764C-3AE8951AF270
 Boot ID:			11f7da06-eaae-4ce1-ac8f-a68df50588dd
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://1.11.0
 Kubelet Version:		v1.7.0-alpha.0.1464+a84f100faa927b
 Kube-Proxy Version:		v1.7.0-alpha.0.1464+a84f100faa927b
PodCIDR:			10.244.1.0/24
ExternalID:			/vcqaDC/vm/kubernetes/node1
Non-terminated Pods:		(3 in total)
  Namespace			Name						CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----						------------	----------	---------------	-------------
  kube-system			heapster-v1.2.0-867844254-gs79h			130m (3%)	130m (3%)	230Mi (2%)	230Mi (2%)
  kube-system			kube-proxy-1n9kw				100m (2%)	0 (0%)		0 (0%)		0 (0%)
  kube-system			kubernetes-dashboard-1019458639-k04rr		0 (0%)		0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  230m (5%)	130m (3%)	230Mi (2%)	230Mi (2%)
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  19m		19m		1	{kubelet node1}				Normal		Starting		Starting kubelet.
  19m		19m		1	{kubelet node1}				Warning		ImageGCFailed		unable to find data for container /
  19m		19m		1	{kubelet node1}				Normal		NodeHasSufficientDisk	Node node1 status is now: NodeHasSufficientDisk
  19m		19m		1	{kubelet node1}				Normal		NodeHasSufficientMemory	Node node1 status is now: NodeHasSufficientMemory
  19m		19m		1	{kubelet node1}				Normal		NodeHasNoDiskPressure	Node node1 status is now: NodeHasNoDiskPressure
  19m		19m		1	{kubelet node1}				Warning		Rebooted		Node node1 has been rebooted, boot id: 11f7da06-eaae-4ce1-ac8f-a68df50588dd
  19m		19m		1	{kube-proxy node1}			Normal		Starting		Starting kube-proxy.


Name:			node2
Role:			
Labels:			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node2
Taints:			<none>
CreationTimestamp:	Thu, 23 Mar 2017 00:17:47 -0700
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------	-----------------			------------------			------			-------
  OutOfDisk 		False 	Thu, 23 Mar 2017 10:02:46 -0700 	Thu, 23 Mar 2017 09:45:38 -0700 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Thu, 23 Mar 2017 10:02:46 -0700 	Thu, 23 Mar 2017 09:45:38 -0700 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Thu, 23 Mar 2017 10:02:46 -0700 	Thu, 23 Mar 2017 09:45:38 -0700 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Thu, 23 Mar 2017 10:02:46 -0700 	Thu, 23 Mar 2017 09:45:38 -0700 	KubeletReady 		kubelet is posting ready status
Addresses:		172.1.16.1,node2,fe80::250:56ff:fe8a:5265
Capacity:
 cpu:		4
 memory:	8170804Ki
 pods:		110
Allocatable:
 cpu:		4
 memory:	8068404Ki
 pods:		110
System Info:
 Machine ID:			8e025a21a4254e11b028584d9d8b12c4
 System UUID:			420A0BE9-DD43-5E93-7B3F-BA9C3B31097B
 Boot ID:			a1b24a9f-8943-4779-b0a1-fa9b3d5d15d0
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://1.11.0
 Kubelet Version:		v1.7.0-alpha.0.1464+a84f100faa927b
 Kube-Proxy Version:		v1.7.0-alpha.0.1464+a84f100faa927b
PodCIDR:			10.244.3.0/24
ExternalID:			/vcqaDC/vm/kubernetes/node2
Non-terminated Pods:		(2 in total)
  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----				------------	----------	---------------	-------------
  kube-system			kube-dns-v19-jf7x1		110m (2%)	0 (0%)		120Mi (1%)	220Mi (2%)
  kube-system			kube-proxy-18fhr		100m (2%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  210m (5%)	0 (0%)		120Mi (1%)	220Mi (2%)
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  9h		18m		15546	{kubelet node2}				Normal		NodeHasSufficientDisk	Node node2 status is now: NodeHasSufficientDisk
  9h		18m		15546	{kubelet node2}				Normal		NodeHasSufficientMemory	Node node2 status is now: NodeHasSufficientMemory
  9h		18m		15546	{kubelet node2}				Normal		NodeHasNoDiskPressure	Node node2 status is now: NodeHasNoDiskPressure
  9h		18m		15544	{kubelet node2}				Normal		NodeReady		Node node2 status is now: NodeReady
  17m		17m		1	{kubelet node2}				Normal		Starting		Starting kubelet.
  17m		17m		1	{kubelet node2}				Warning		ImageGCFailed		unable to find data for container /
  17m		17m		1	{kube-proxy node2}			Normal		Starting		Starting kube-proxy.
  17m		17m		17	{kubelet node2}				Normal		NodeHasSufficientDisk	Node node2 status is now: NodeHasSufficientDisk
  17m		17m		17	{kubelet node2}				Normal		NodeHasSufficientMemory	Node node2 status is now: NodeHasSufficientMemory
  17m		17m		17	{kubelet node2}				Normal		NodeHasNoDiskPressure	Node node2 status is now: NodeHasNoDiskPressure
  17m		17m		16	{kubelet node2}				Warning		Rebooted		Node node2 has been rebooted, boot id: a1b24a9f-8943-4779-b0a1-fa9b3d5d15d0
  17m		17m		16	{kubelet node2}				Normal		NodeReady		Node node2 status is now: NodeReady


Name:			node3
Role:			
Labels:			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node3
Taints:			<none>
CreationTimestamp:	Thu, 23 Mar 2017 00:17:46 -0700
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------	-----------------			------------------			------			-------
  OutOfDisk 		False 	Thu, 23 Mar 2017 10:02:49 -0700 	Thu, 23 Mar 2017 09:45:59 -0700 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Thu, 23 Mar 2017 10:02:49 -0700 	Thu, 23 Mar 2017 09:45:59 -0700 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Thu, 23 Mar 2017 10:02:49 -0700 	Thu, 23 Mar 2017 09:45:59 -0700 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Thu, 23 Mar 2017 10:02:49 -0700 	Thu, 23 Mar 2017 09:45:59 -0700 	KubeletReady 		kubelet is posting ready status
Addresses:		172.1.7.1,node3,fe80::250:56ff:fe8a:1a25
Capacity:
 cpu:		4
 memory:	8170804Ki
 pods:		110
Allocatable:
 cpu:		4
 memory:	8068404Ki
 pods:		110
System Info:
 Machine ID:			8e025a21a4254e11b028584d9d8b12c4
 System UUID:			420ABA54-F763-A3B6-97D9-1977648A4BCA
 Boot ID:			67d22a0a-5d56-4545-8b2f-facaf8c3765b
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://1.11.0
 Kubelet Version:		v1.7.0-alpha.0.1464+a84f100faa927b
 Kube-Proxy Version:		v1.7.0-alpha.0.1464+a84f100faa927b
PodCIDR:			10.244.2.0/24
ExternalID:			/vcqaDC/vm/kubernetes/node3
Non-terminated Pods:		(2 in total)
  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----				------------	----------	---------------	-------------
  kube-system			kube-dns-v19-1g47f		110m (2%)	0 (0%)		120Mi (1%)	220Mi (2%)
  kube-system			kube-proxy-s7hf8		100m (2%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  210m (5%)	0 (0%)		120Mi (1%)	220Mi (2%)
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  9h		17m		16122	{kubelet node3}				Normal		NodeHasSufficientDisk	Node node3 status is now: NodeHasSufficientDisk
  9h		17m		16122	{kubelet node3}				Normal		NodeHasSufficientMemory	Node node3 status is now: NodeHasSufficientMemory
  9h		17m		16122	{kubelet node3}				Normal		NodeHasNoDiskPressure	Node node3 status is now: NodeHasNoDiskPressure
  9h		17m		16120	{kubelet node3}				Normal		NodeReady		Node node3 status is now: NodeReady
  17m		17m		1	{kubelet node3}				Normal		Starting		Starting kubelet.
  17m		17m		1	{kubelet node3}				Warning		ImageGCFailed		unable to find data for container /
  17m		17m		1	{kube-proxy node3}			Normal		Starting		Starting kube-proxy.
  17m		16m		17	{kubelet node3}				Normal		NodeHasSufficientDisk	Node node3 status is now: NodeHasSufficientDisk
  17m		16m		17	{kubelet node3}				Normal		NodeHasSufficientMemory	Node node3 status is now: NodeHasSufficientMemory
  17m		16m		17	{kubelet node3}				Normal		NodeHasNoDiskPressure	Node node3 status is now: NodeHasNoDiskPressure
  17m		16m		16	{kubelet node3}				Warning		Rebooted		Node node3 has been rebooted, boot id: 67d22a0a-5d56-4545-8b2f-facaf8c3765b
  17m		16m		16	{kubelet node3}				Normal		NodeReady		Node node3 status is now: NodeReady


Name:			node4
Role:			
Labels:			beta.kubernetes.io/arch=amd64
			beta.kubernetes.io/os=linux
			kubernetes.io/hostname=node4
Taints:			<none>
CreationTimestamp:	Thu, 23 Mar 2017 00:18:13 -0700
Phase:			
Conditions:
  Type			Status	LastHeartbeatTime			LastTransitionTime			Reason			Message
  ----			------	-----------------			------------------			------			-------
  OutOfDisk 		False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 09:47:08 -0700 	KubeletHasSufficientDisk 	kubelet has sufficient disk space available
  MemoryPressure 	False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 09:47:08 -0700 	KubeletHasSufficientMemory 	kubelet has sufficient memory available
  DiskPressure 		False 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 09:47:08 -0700 	KubeletHasNoDiskPressure 	kubelet has no disk pressure
  Ready 		True 	Thu, 23 Mar 2017 10:02:47 -0700 	Thu, 23 Mar 2017 09:47:08 -0700 	KubeletReady 		kubelet is posting ready status
Addresses:		node4,172.1.46.1,fe80::250:56ff:fe8a:f5f0
Capacity:
 cpu:		4
 memory:	8170804Ki
 pods:		110
Allocatable:
 cpu:		4
 memory:	8068404Ki
 pods:		110
System Info:
 Machine ID:			8e025a21a4254e11b028584d9d8b12c4
 System UUID:			420A87C1-7FA9-4917-D42A-FE90DA7F5B53
 Boot ID:			b5a0b122-d0df-4018-9b02-e4cb0c7f4a27
 Kernel Version:		4.4.8-esx
 OS Image:			Debian GNU/Linux 8 (jessie)
 Operating System:		linux
 Architecture:			amd64
 Container Runtime Version:	docker://1.11.0
 Kubelet Version:		v1.7.0-alpha.0.1464+a84f100faa927b
 Kube-Proxy Version:		v1.7.0-alpha.0.1464+a84f100faa927b
PodCIDR:			10.244.4.0/24
ExternalID:			/vcqaDC/vm/kubernetes/node4
Non-terminated Pods:		(2 in total)
  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ---------			----				------------	----------	---------------	-------------
  default			thinclaimpod			0 (0%)		0 (0%)		0 (0%)		0 (0%)
  kube-system			kube-proxy-8wrhw		100m (2%)	0 (0%)		0 (0%)		0 (0%)
Allocated resources:
  (Total limits may be over 100 percent, i.e., overcommitted.
  CPU Requests	CPU Limits	Memory Requests	Memory Limits
  ------------	----------	---------------	-------------
  100m (2%)	0 (0%)		0 (0%)		0 (0%)
Events:
  FirstSeen	LastSeen	Count	From			SubObjectPath	Type		Reason			Message
  ---------	--------	-----	----			-------------	--------	------			-------
  9h		17m		16207	{kubelet node4}				Normal		NodeHasSufficientDisk	Node node4 status is now: NodeHasSufficientDisk
  9h		17m		16207	{kubelet node4}				Normal		NodeHasSufficientMemory	Node node4 status is now: NodeHasSufficientMemory
  9h		17m		16207	{kubelet node4}				Normal		NodeHasNoDiskPressure	Node node4 status is now: NodeHasNoDiskPressure
  9h		17m		16205	{kubelet node4}				Normal		NodeReady		Node node4 status is now: NodeReady
  16m		16m		1	{kubelet node4}				Normal		Starting		Starting kubelet.
  16m		16m		1	{kubelet node4}				Warning		ImageGCFailed		unable to find data for container /
  16m		16m		1	{kube-proxy node4}			Normal		Starting		Starting kube-proxy.
  16m		15m		7	{kubelet node4}				Normal		NodeHasSufficientDisk	Node node4 status is now: NodeHasSufficientDisk
  16m		15m		7	{kubelet node4}				Normal		NodeHasSufficientMemory	Node node4 status is now: NodeHasSufficientMemory
  16m		15m		7	{kubelet node4}				Normal		NodeHasNoDiskPressure	Node node4 status is now: NodeHasNoDiskPressure
  16m		15m		6	{kubelet node4}				Warning		Rebooted		Node node4 has been rebooted, boot id: b5a0b122-d0df-4018-9b02-e4cb0c7f4a27
  16m		15m		6	{kubelet node4}				Normal		NodeReady		Node node4 status is now: NodeReady
$

```

@tusharnt can we change template file in the nightly CI job for master to use ipv4 addresses only until this issue is resolved?

**Comment by tusharnt** [comment]
@divyenpatel Sounds good to me. As we discussed offline, please update kubernetes#42690 with these findings as well.

**Comment by BaluDontu** [comment]
When vsphere drops all IPV6 addresses and only allow IPV4 addresses when returning from NodeAddresses() inside cloud Provider, it is working perfectly fine and I don't see this problem at all.

So, I think its the problem with Kubernetes core, to handle the IPV6 addresses. Hence we see this issue. Please check my prototype PR where I drop IPV6 addresses in NodeAddresses() - vmware#138.

Will work @dhchen on this.

**Comment by dhchen** [comment]
@BaluDontu It seems that you mentioned the wrong guy since I know nothing about kubernetes

**Comment by BaluDontu** [comment]
Sorry its @dchen1107 ..

**Comment by BaluDontu** [comment]
I have PR out for review - https://github.com/kubernetes/kubernetes/pull/45181

**Comment by BaluDontu** [comment]
The PR is merged. Closing it now.
